doc/netplan-yaml.md

```diff
@@ -491,7 +491,10 @@ Match devices by MAC when setting options like: `wakeonlan` or `*-offload`.
 - **`macaddress`** (scalar)
 
   > Set the device's MAC address. The MAC address must be in the form
-  > `XX:XX:XX:XX:XX:XX`.
+  > "XX:XX:XX:XX:XX:XX". When using `NetworkManager` as renderer, the following
+  > values are also accepted: `preserve`, `permanent`, `random` and `stable`.
+  > When using `networkd`, the following values are also accepted: `persistent`,
+  > `random`.
```

Comment:
IIUC networkd's `persistent` is the same as NetworkManager's `permanent` [1,2], where "permanent MAC" is the more common terminology IMO.

We should probably combine persistent+permanent into a single "permanent" Netplan value. Also "random" is the same for both backend anyway. So we'd only need to specify the `preserve` and `stable` special values for NM, everything else could be shared. Alternatively, "persistent" could be a (secondary) alias to "permanent".

We want to reduce such differences and make Netplan do the right thing, depending on the chosen renderer.

[1] https://developer-old.gnome.org/NetworkManager/stable/settings-802-3-ethernet.html
[2] https://www.freedesktop.org/software/systemd/man/latest/systemd.link.html#MACAddressPolicy=

Reply:
I'm using `permanent` for both renderers now.